staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/status_test.go

```diff
@@ -85,3 +86,29 @@ func TestAPIStatus(t *testing.T) {
 		}
 	}
 }
+
+func TestUnexpectedErrorToAPIStatus(t *testing.T) {
```

Comment:
Can we combine this test case with the already `TestAPIStatus` present to avoid code duplication and for consistency?